mobile/src/main/java/ch/epfl/sdp/mobile/application/tournaments/StoreDocumentTournament.kt

```diff
@@ -115,7 +119,7 @@ class StoreDocumentTournament(
     runCatching {
       val results =
           store
-              .collection("games")
+              .collection(ChessDocument.Collection)
               .whereEquals("tournamentId", reference.uid)
               .whereNotEquals("poolId", null)
```

Comment:
Missing field.

Reply:
Added in 715a39794e4fd9228eaaf11ad0bf8a1682f5edfd